core/src/test/scala/integration/kafka/tools/TestMirrorMaker.scala

```diff
@@ -0,0 +1,69 @@
+package kafka.tools
+
+import java.util.Properties
+
+import kafka.integration.KafkaServerTestHarness
+import kafka.server.KafkaConfig
+import kafka.tools.MirrorMaker.{MirrorMakerNewConsumer, MirrorMakerProducer}
+import kafka.utils.TestUtils
+import org.apache.kafka.clients.consumer.{ConsumerConfig, KafkaConsumer}
+import org.apache.kafka.clients.producer.{ProducerConfig, ProducerRecord}
+import org.apache.kafka.common.serialization.ByteArrayDeserializer
+import org.junit.Assert.assertTrue
+import org.junit.{After, Before, Test}
+
+class TestMirrorMaker extends KafkaServerTestHarness {
```

Comment:
Ah, that makes sense because they both have the same package (it's just the source folder that differs). In that case, I suggest `MirrorMakerIntegrationTest` for this one so the difference is obvious.